txrelayer/txrelayer.go

```diff
@@ -130,6 +135,12 @@ func (t *TxRelayerImpl) sendTransactionLocked(txn *ethgo.Transaction, key ethgo.
 		return ethgo.ZeroHash, err
 	}
 
+	if t.writer != nil {
+		_, _ = t.writer.Write([]byte(
+			fmt.Sprintf("[ROOTCHAIN - CONTRACTS DEPLOYMENT TRANSACTION DETAIL]\nFrom = %s \nGas = %d  \nGas Price = %d\n",
```

Comment:
Removing extra space there:
```suggestion
			fmt.Sprintf("[ROOTCHAIN - CONTRACTS DEPLOYMENT TRANSACTION DETAIL]\nFrom = %s \nGas = %d \nGas Price = %d\n",
```